docs/hooks.md

```diff
@@ -0,0 +1,242 @@
+# Introduction
+Migration hooks provide a means for running custom code at points in the migration to handle logic unique to a migration. This may include writing new configuration files or installing additional packages in preparation for migration to prepare for the new environment.
+
+# Default hook image
+The default hook image is `quay.io/konveyor/hook-runner:latest`. It is based off of the Ansible Runner image with the addition of python-openshift to provided Ansible k8s resources as well as a recent oc binary.
+
+# Hook execution
+When an Ansible playbook is provided as part of a migration hook it will be mounted into the hook container as a ConfigMap. In either case the hook container will be run as job on the desired cluster, using the default ServiceAccount in the konveyor-forklift namespace.
```

Comment:
Or, the user may specify a _serviceAccount_ in the Hook specification.

Reply:
We should probably recommend they set the SA on the Hook to `serviceAccount: forklift-controller`.